core/bindings.rs

```diff
@@ -36,84 +38,89 @@ const UNDEFINED_OP_ID_MSG: &str =
-        function: serialize.map_fn_to(),
-      },
-      v8::ExternalReference {
-        function: deserialize.map_fn_to(),
-      },
-      v8::ExternalReference {
-        function: get_promise_details.map_fn_to(),
-      },
-      v8::ExternalReference {
-        function: get_proxy_details.map_fn_to(),
-      },
-      v8::ExternalReference {
-        function: is_proxy.map_fn_to(),
-      },
-      v8::ExternalReference {
-        function: memory_usage.map_fn_to(),
-      },
-      v8::ExternalReference {
-        function: call_console.map_fn_to(),
-      },
-      v8::ExternalReference {
-        function: set_wasm_streaming_callback.map_fn_to(),
-      },
-    ])
-  });
-
+pub fn external_references(isolate_ptr: *mut c_void) -> v8::ExternalReferences {
+  v8::ExternalReferences::new(&[
+    v8::ExternalReference {
+      function: opcall_async.map_fn_to(),
+    },
+    v8::ExternalReference {
+      function: opcall_sync.map_fn_to(),
+    },
+    v8::ExternalReference {
+      function: ref_op.map_fn_to(),
+    },
+    v8::ExternalReference {
+      function: unref_op.map_fn_to(),
+    },
+    v8::ExternalReference {
+      function: set_macrotask_callback.map_fn_to(),
+    },
+    v8::ExternalReference {
+      function: set_nexttick_callback.map_fn_to(),
+    },
+    v8::ExternalReference {
+      function: set_promise_reject_callback.map_fn_to(),
+    },
+    v8::ExternalReference {
+      function: set_uncaught_exception_callback.map_fn_to(),
+    },
+    v8::ExternalReference {
+      function: run_microtasks.map_fn_to(),
+    },
+    v8::ExternalReference {
+      function: has_tick_scheduled.map_fn_to(),
+    },
+    v8::ExternalReference {
+      function: set_has_tick_scheduled.map_fn_to(),
+    },
+    v8::ExternalReference {
+      function: eval_context.map_fn_to(),
+    },
+    v8::ExternalReference {
+      function: queue_microtask.map_fn_to(),
+    },
+    v8::ExternalReference {
+      function: create_host_object.map_fn_to(),
+    },
+    v8::ExternalReference {
+      function: encode.map_fn_to(),
+    },
+    v8::ExternalReference {
+      function: decode.map_fn_to(),
+    },
+    v8::ExternalReference {
+      function: serialize.map_fn_to(),
+    },
+    v8::ExternalReference {
+      function: deserialize.map_fn_to(),
+    },
+    v8::ExternalReference {
+      function: get_promise_details.map_fn_to(),
+    },
+    v8::ExternalReference {
+      function: get_proxy_details.map_fn_to(),
+    },
+    v8::ExternalReference {
+      function: is_proxy.map_fn_to(),
+    },
+    v8::ExternalReference {
+      function: memory_usage.map_fn_to(),
+    },
+    v8::ExternalReference {
+      function: call_console.map_fn_to(),
+    },
+    v8::ExternalReference {
+      function: set_wasm_streaming_callback.map_fn_to(),
+    },
+    #[cfg(feature = "napi")]
+    v8::ExternalReference {
+      function: napi_open_func.map_fn_to(),
+    },
+    v8::ExternalReference {
+      pointer: isolate_ptr,
+    },
```

Comment:
Nitpick, make it the first reference to avoid scrolling to see where `isolate_ptr` is used